security/sshdo/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                sshdo
+version             1.1
+revision            0
+
+categories          security
+license             GPL-2+
+maintainers         {raf.org:raf @macportsraf} openmaintainer
+description         controls which commands may be executed via incoming ssh
+long_description    {*}${description}. \
+                    \n\nsshdo provides an easily configurable way of controlling which \
+                    commands may be executed via incoming ssh connections by specific \
+                    users and specific keys. Training mode allows a key to be used for \
+                    any command, while logging them. Then sshdo can learn from the logs \
+                    to know what commands to allow. It can also unlearn commands that \
+                    are no longer in use. This mitigates against private ssh key compromise \
+                    with very little effort.
+
+github.setup        raforg ${name} ${version} v
+github.tarball_from releases
+checksums           rmd160 6de62e1182526b5c245b6b7a75974fffd0ffeda1 \
+                    sha256 04a36696538ddbf188f86c357c1d2eaca21b2a09dd555bae072ed8660400c75a \
+                    size 78652
+
+patchfiles          macports.patch
+
+# There is nothing to build
+use_configure       no
+build.target        help
```

Comment:
If your intent is to have `make` do nothing by just running a useless make target, you can simply use:

```
build {}
```

This will effectively disable the build phase.

Reply:
Thanks. Done.

Reply:
The `configure` phase is separate from the `build` phase, so you may still need `use_configure no` alongside `build {}`.